src/realm/parser/driver.cpp

```diff
@@ -1355,6 +1280,136 @@ std::unique_ptr<Subexpr> ConstantNode::visit(ParserDriver* drv, DataType hint)
     return ret;
 }
 
+std::vector<std::shared_ptr<Subexpr>> ConstantNode::clone_list_of_args(std::vector<Mixed>& mixed_args)
+{
+    std::vector<std::shared_ptr<Subexpr>> args_in_list;
+    args_in_list.reserve(mixed_args.size());
+    std::shared_ptr<Subexpr> ret;
+
+    for (const auto& mixed : mixed_args) {
+        if (!mixed.is_null()) {
```

Comment:
Null isn't being handled correctly here (it is inserting the last moved value of `ret` into the list).